### DISCUSSION TITLE
How can I add multiple files from `Add File` dialog?

### DISCUSSION THREAD

**Discussion by mahmoudajawad** [yetone/avante.nvim#1012] [Q&A] [answered]
When I hit `@` in the files section, I'm given a file picker where I can select a file to add to chat. Since this is telescope, I can select multiple files with `Tab`, but I couldn't figure out how can I add all selected files from file picker to chat.

**Comment by dogonthehorizon** [comment, marked as answer]
There's a PR open to add this functionality:
https://github.com/yetone/avante.nvim/pull/971

**Reply by mahmoudajawad** [reply]
Thanks. I thought I searched issues well, but missed this one.
